src/styles.scss

```diff
@@ -1 +1,10 @@
 /* You can add global styles to this file, and also import other style files */
+
```

Comment:
Why the extra line? 🗡

Reply:
lint approved, also css formatter inserts too